src/Documentation/SidebarMenu/helper.test.js

```diff
@@ -0,0 +1,329 @@
+describe('SidebarMenu/helper', () => {
```

Comment:
is a regular convention to have tests along the actual implementation? it was always easier for me to manage them in separate global directory (that reflects the structure of actual code base)

Reply:
It depends on the team, but in my experience it is more convenient to store all component's parts in one folder:

- Easier to rename and move components.
- If you want to delete component - you just remove it's folder with code, styles, assets, locales, tests and docs. No need to go to multiple folders to delete everything.
- All related files is grouped in FS.
- Etc.

Reply:
ok, let's give it a try 👍